bundles/core/pom.xml

```diff
@@ -366,7 +368,12 @@
             <artifactId>uber-jar</artifactId>
             <classifier>apis</classifier>
         </dependency>
-
+        <!-- https://mvnrepository.com/artifact/org.jetbrains/annotations -->
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>RELEASE</version>
```

Comment:
Please use a fixed version here. Please also check https://github.com/adobe/aem-core-cif-components/pull/89#discussion_r304249641 and see if `javax.annotation` provides an equal annotation.

Reply:
thanks, using javax.annotation.Nonnull instead, dep removed.